src/components/Stations.jsx

```diff
@@ -1,93 +1,87 @@
 import React from 'react';
-import Button from '@material-ui/core/Button';
-import Menu from '@material-ui/core/Menu';
-import MenuItem from '@material-ui/core/MenuItem';
-import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
-import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+// import Button from '@material-ui/core/Button';
+// import InputLabel from '@material-ui/core/InputLabel';
+// import FormHelperText from '@material-ui/core/FormHelperText';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+// import NativeSelect from '@material-ui/core/NativeSelect';
+
+const useStyles = makeStyles((theme) => ({
+  formControl: {
+    fontFamily: "Roboto Mono",
+    fontSize: '16px',
+    lineHeight: '16px',
+    fontWeight: '500'
+  },
+  selectEmpty: {
+    marginTop: theme.spacing(2),
+  },
+}));
 
 
 const options = [
-  'ALL',
-  'ARCHIVE',
-  'AR',
-  'AUS',
-  'CAMS',
-  'EXOSS',
-  'GMN',
-  'MA',
-  'NCA',
-  'TEXAS',
-  'UAE',
-  'BENELUX',
-  'CHILE',
-  'EDMOND',
-  'FL',
-  'LOCAMS',
-  'NAMIBIA',
-  'NZ',
-  'SA',
-  'SONOTACO',
-  'TK'
+    'ALL LOCATIONS',
```

Comment:
We shall probably considering using a mapping that translation site id to it's name in the dropdown?